www/src/context/SettingsContext.jsx

```diff
@@ -11,12 +11,27 @@ const defaultValue = {
 const SettingsContext = createContext(defaultValue);
 
 const SettingsContextProvider = ({ children }) => {
+  // gatsby does not have access to the localStorage during the build (and first render)
+  // so sadly we cannot initialize theme with value from localStorage
```

Comment:
The result of this tradeoff is the default Open edX theme will flash briefly before the edX.org theme is applied. I think this should be OK.

Though, I wonder if cookies would help get around this? Probably not since it'd still need access to `document` after the first render, too.

Reply:
Yeah, I don't think there is a way to get around this, we can't access anything on the first render